src/librustc_codegen_ssa/mir/block.rs

```diff
@@ -141,6 +141,22 @@ impl<'a, 'tcx> TerminatorCodegenHelper<'tcx> {
             }
 
             if let Some((ret_dest, target)) = destination {
+                if bx.tcx().sess.opts.cg.tracer.sir_labels() &&
+                    bx.tcx().def_path_str(fx.instance.def_id()) != "core::intrinsics::drop_in_place" &&
+                        bx.tcx().def_path_str(fx.instance.def_id()) != "std::intrinsics::drop_in_place" &&
+                        bx.tcx().def_path_str(fx.instance.def_id()) != "ptr::drop_in_place" &&
+                        bx.cx().tcx().symbol_name(fx.instance).name.as_str() != "main" &&
+                        !bx.tcx().crate_name(LOCAL_CRATE).as_str().starts_with("rustc") {
+                            let llbb = bx.llbb();
+                            use ykpack::BLOCK_LABEL_PREFIX;
```

Comment:
Do we need another prefix for this kind of label?

Reply:
I guess we could but it's not strictly necessary.

Reply:
If a block contains a call to a function with a return value, won't we insert two labels of the same name at different places?

I may be wrong, I don't 100% remember the details.

Reply:
My gut feeling says no, but I'm not sure. Let's rename it just in case.

Reply:
Ah damn. Because I moved the label generation into `BuilderCx` I now need to add another argument to the `add_yk_block_label` function.